mitodl/micromasters
# Add validation for recipient variable tags

#### What are the relevant tickets?
Fixes #3164 

#### What's this PR do?
Adds browser validation for recipient variable tags which are broken up by markup

#### How should this be manually tested?
In the learners page open the email dialog. Insert a recipient variable into the text. Select a part of the variable and make it bold. Submit the email and verify that it won't let you send it.
